DEV-1/DEV-1/SerialIdenticalSymbols.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace DEV_1
+{
+    /// <summary>
+    /// This class contains methods to calculate number of serial identical symbols.
+    /// </summary>
+    class SerialIdenticalSymbols
+    {
+        /// <summary>
+        /// This class calculates number of serial identical symbols.
+        /// </summary>
+        /// <param name="theString">The main string.</param>
+        /// <returns>
+        /// Returns maximum number of serial identical symbols.
+        /// </returns>
+        public int GetMaxNumberOfSerialIdenticalSymbols(string theString)
```

Comment:
what if theString is empty? I suppose your implementation returns 1 that is wrong

Reply:
my implementation returns "Wrong number of arguments." as it supposed to be

Reply:
Obviously this method never returns "Wrong number of arguments." as it can return only int